workspace-manager-clienttests/src/main/resources/uploadlists/TerraWSMDev.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "TerraWSMDev",
+  "description": "Upload to the broad-dsde-dev project, testrunner-results bucket",
+  "uploaderServiceAccountFile": "dev-testrunner-sa.json",
+  "uploadScripts": [
+    {
+      "name": "CompressDirectoryToBucket",
+      "description": "Save the compressed results directory to a bucket.",
+      "parameters": ["gs://broad-dsde-dev-testrunner-results"]
```

Comment:
Per the README above, should this bucket path include workspace-manager in its name?

Reply:
I intentionally leave it this way. Will have `devOps` review the bucket path and OK it. So right now I am simply setting this up adhoc.